src/main/resources/lib/advisor/validateOnLoad.jelly

```diff
@@ -0,0 +1,39 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:d="jelly:define" xmlns:st="jelly:stapler">
+  <st:documentation>
+    Check if the current admin is connected to CloudBees Jenkins Advisor.
+    <st:attribute name="email" use="required">
+      Email of the admin.
+    </st:attribute>
+    <st:attribute name="password" use="required">
+      Password of the admin.
+    </st:attribute>
+  </st:documentation>
+
+  <style type="text/css">
+    .alert-success {
+      color: #155724;
+      background-color: #d4edda;
+      border-color: #c3e6cb;
+    }
+  </style>
+
+  <div>
+    <j:if test="${email!=null and password!=null}"> 
+      <j:set var="validationStatus" value="${app.getDescriptorByName('com.cloudbees.jenkins.plugins.advisor.AdvisorGlobalConfiguration').connectionTest(email+','+password)}"/>
+        <j:if test="${validationStatus.contains('connected')}">
+          <div class="alert alert-success" role="alert">
+            <h4 class="alert-heading">${validationStatus}</h4>
+            <p>If you change your credentials, please check using "Test Connection" button below.</p>
```

Comment:
> If you change your credentials, please check using "Test Connection" button below

Grammer:

`change` -> `changed` ?

Reply:
Since it was at the top of the page before the configuration option, I thought that it would make sense to refer to the user's (potential) future action.

Reply:
ok